src/drivers/Kconfig

```diff
@@ -46,4 +46,15 @@ config IMX_IRQSTEER
 
 	  If unsure, select "n".
 
+config IMX_SDMA
+	bool "Enable i.MX SDMA controller support"
```

Comment:
You are adding IMX_SDMA in this patch. And then later you are moving it in another location. Can you add it directly to src/drivers/imx/Kconfig?

Sorry but this is the correct way to do it.